package-lock.json

```diff
@@ -5431,6 +5345,7 @@
       "version": "7.0.1",
       "resolved": "https://registry.npmjs.org/read-package-json/-/read-package-json-7.0.1.tgz",
       "integrity": "sha512-8PcDiZ8DXUjLf687Ol4BR8Bpm2umR7vhoZOzNRt+uxD9GpBh/K+CAAALVIiYFknmvlmyg7hM7BSNUXPaCCqd0Q==",
+      "deprecated": "This package is no longer supported. Please use @npmcli/package-json instead.",
```

Comment:
Do we need to action anything here

Reply:
We don't use https://www.npmjs.com/package/read-package-json directly, so I guess this is pulled by a 3rd party package. I don't think this is a blocker.